examples/sdk-usage/src/main/java/io/opentelemetry/sdk/example/BasicExample.java

```diff
@@ -0,0 +1,44 @@
+package io.opentelemetry.sdk.example;
+
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdk;
+import io.opentelemetry.sdk.trace.TracerSdkFactory;
+
+/**
+ * This example shows how to instantiate an OpenTelemetry tracer.
+ */
+class BasicExample {
+
+	public static void main(String[] args) throws InterruptedException {
+		// Get the tracer factory. Multiple tracer can be instantiated.
+		TracerSdkFactory tracerProvider = OpenTelemetrySdk.getTracerFactory();
```

Comment:
Formatting looks non-standard in these file. Also, you're missing the copyright notices.